### PR TITLE
Update deps and Elixir version

### PR DESCRIPTION
**Pull request by rlonstein**
* Update the deps to current release(s) for Elixir
* Fix syntax in tests broken by dep changes
* quiet ambiguity warnings in tests

Tested with:
* Ubuntu 16.04.1, Erlang/OTP 20, Elixir 1.5.2
* Ubuntu 16.04.1, Erlang/OTP 19, Elixir 1.4.5

### CODE REVIEW

**Comment by a-type** [comment]
Hi @rlonstein, thanks for the update! I believe we've stopped using Coveralls, so it needs to be removed from the build.

File: mix.exs [hunk 1]
```diff
@@ -29,23 +29,23 @@ defmodule Bandwidth.Mixfile do
 
   defp package do
     [ files: ["lib", "mix.exs", "README.md", "LICENSE.md"],
-      contributors: ["Tyler Cross"],
+      contributors: ["Tyler Cross", "Ross Lonstein"],
       licenses: ["MIT"],
       links: %{
-        GitHub: "https://github.com/wtcross/elixir-bandwidth",
+        GitHub: "https://github.com/rlonstein/elixir-bandwidth",
```
Comment (dtolb) [inline mix.exs]: Pardon my ignorance, is there any reason that this _shouldn't_ be `https://github.com/Bandwidth/elixir-bandwidth/` ? I'm not too hip on elixir package management, so I could be way off base.

Regardless, thank you so much for the update!

Comment (rlonstein) [reply mix.exs]: No reason, was trying to get the coveralls.io working and swapped out the original author's repo for my own. I'll switch that to your upstream one.

Comment (dtolb) [reply mix.exs]: gotcha! Thank you for the feedback.

Comment (rlonstein) [reply mix.exs]: Stripped out coveralls.

**Pull request merged**